book/03-git-branching/sections/branch-management.asc

```diff
@@ -73,3 +73,42 @@ $ git branch --no-merged master
   featureB
 ----
 ====
+
+==== Changing a branch name
+
+You can change a branch name with the `git branch -m` command.
+If you want to collaborate with others, on the newly changed branch, you'll need to push it.
+
+[source,console]
+----
+$ git branch -m bad-branch-name corrected-branch-name
+$ git push -u origin corrected-branch-name
+----
+
+===== Changing the master branch name
+
+[WARNING]
+====
+Changing the name of a branch like master/main/mainline/default will break the integrations, services, helper utilities and build/release scripts that your repository uses.
+Read the entire section before making any changes.
+Do not attempt this move without first consulting with your co-workers.
+Check beforehand where the old branch name is used in your project by doing a thorough search.
+====
+
+Changing the name from _master_ to _main_ can be done with the `git branch -m` command.
+After changing the name, you'll need to push the renamed branch so that others can see it too.
+Other users with a clone/fork of your repository will need to change their default branch as well.
+
+[source,console]
+----
+$ git branch -m master main
+$ git push -u origin main
+----
+
+Things to update after pushing the new default branch:
```

Comment:
This feels a tiny bit short. Maybe a little more?

```suggestion
That changes the branch name in your repo, and makes the new name available in the remote.
Now you have a few more tasks in front of you to complete the transition:
```

Reply:
Yes, that's better than what I made. In the commit it goes! 😄